CMakeLists.txt

```diff
@@ -77,6 +77,7 @@ if(CMAKE_SOURCE_DIR STREQUAL CMAKE_CURRENT_SOURCE_DIR)
   # to import lots of dependencies from AddMLIRPython to make this work.
   set(MLIR_ENABLE_BINDINGS_PYTHON 1)
   option(TORCH_MLIR_ENABLE_JIT_IR_IMPORTER "Enables JIT IR Importer" ON)
+  option(TORCH_MLIR_ENABLE_EAGER_MODE "Enables Eager Mode" ON)
```

Comment:
I think we should have this always on. It doesn't have any new dependency or anything, right? (it doesn't even seem to have C++ code)